src/platform/packages/shared/kbn-esql-ast/src/commands_registry/commands/eval/validate.test.ts

```diff
@@ -47,35 +47,6 @@ describe('EVAL Validation', () => {
     evalExpectErrors('from a_index | eval col0=["a", "b"]', []);
     evalExpectErrors('from a_index | eval col0=null', []);
   });
-  test('date_diff', () => {
```

Comment:
No need to test specific functions. These scenarios are covered in our generic functions test file.